app/components/container-logs/component.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend({
       let ansiup = new AnsiUp.default;
       set(this, 'status', 'connected');
       var isFollow = ($body.scrollTop() + $body.outerHeight() + 10) >= body.scrollHeight;
-      const data = decodeURIComponent(AWS.util.base64.decode(message.data));
```

Comment:
AFAIR the 1.x shell used to encode the data on the server-side;  Since the k8s server isn't, shouldn't it just be removed?  decode(encode()) sounds like no-op.

Reply:
Yes! You are right. Thanks